### PR TITLE
Update dependencies (#227)

### PR DESCRIPTION
**Pull request by bedrich-schindler**
@adamkudrna For info, there are few incompatibilities that do not allow us to update everything to the latest versions.

```
npm outdated
Package                                Current          Wanted   Latest  Location
@babel/plugin-transform-spread          7.12.1          7.12.1  7.12.13  @react-ui-org/react-ui
babel-core                      7.0.0-bridge.0  7.0.0-bridge.0   6.26.3  @react-ui-org/react-ui
eslint-plugin-markdown              2.0.0-rc.2      2.0.0-rc.2    1.0.2  @react-ui-org/react-ui
postcss-loader                           4.2.0           4.2.0    5.0.0  @react-ui-org/react-ui
react                                  16.14.0         16.14.0   17.0.1  @react-ui-org/react-ui
react-dom                              16.14.0         16.14.0   17.0.1  @react-ui-org/react-ui
sass-loader                             10.1.1          10.1.1   11.0.0  @react-ui-org/react-ui
webpack                                 4.46.0          4.46.0   5.21.2  @react-ui-org/react-ui
webpack-cli                             3.3.12          3.3.12    4.5.0  @react-ui-org/react-ui
```

React 17 cannot be updated due to missing Rect 17 adapter for enzyme (will be released soon), Webpack cannot be updated because Gatsby do not suport Webpack 5. Sass loader and postcss loader are broken, I was not able to fix it.

`@babel/plugin-transform-spread` must have been added to solve Gatsby issues, we need to remove it later when it will be resolved.

### CODE REVIEW

**Comment by adamkudrna** [comment]
> Sass loader and postcss loader are broken, I was not able to fix it.

That's because [both](https://github.com/webpack-contrib/sass-loader/releases/tag/v11.0.0) of [them](https://github.com/webpack-contrib/postcss-loader/releases/tag/v5.0.0) require Webpack 5 in their latest versions.

File: package.json [hunk 1]
```diff
@@ -43,46 +43,47 @@
...
-    "enzyme-adapter-react-16": "^1.15.5",
+    "enzyme-adapter-react-16": "^1.15.6",
     "enzyme-to-json": "^3.6.1",
-    "eslint": "^7.15.0",
+    "eslint": "^7.19.0",
     "eslint-config-airbnb": "^18.2.1",
     "eslint-loader": "^4.0.2",
     "eslint-plugin-import": "^2.22.1",
     "eslint-plugin-jsx-a11y": "^6.4.1",
     "eslint-plugin-markdown": "^2.0.0-rc.0",
```
Comment (adamkudrna) [inline package.json:71]: How about bumping this to RC2?

https://github.com/eslint/eslint-plugin-markdown/releases

File: package.json [hunk 2]
```diff
@@ -43,46 +43,47 @@
...
-    "eslint-plugin-react": "^7.21.5",
+    "eslint-plugin-react": "^7.22.0",
     "eslint-plugin-react-hooks": "^4.2.0",
     "identity-obj-proxy": "^3.0.0",
     "jest": "^26.6.3",
-    "markdownlint-cli2": "0.0.11",
-    "nodemon": "^2.0.6",
-    "postcss": "^8.2.1",
-    "postcss-loader": "^4.1.0",
+    "markdownlint-cli2": "^0.0.13",
```
Comment (adamkudrna) [inline package.json:77]: Let's be careful with 0.x versions, I'd leave out the `^`.

**Review by adamkudrna — APPROVED** [review]
Just two tiny nits and it's ✅ .

File: package.json (hunk 1 above)
Comment (bedrich-schindler) [reply package.json:71]: RC2 is currently installed due to `^`. Can update it.

File: package.json (hunk 2 above)
Comment (bedrich-schindler) [reply package.json:77]: Yep, I overlooked this. Thx

**Pull request merged**